Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:17-jdk-slim
```

Comment:
Why not a never one like 21/22 jdk version?

Reply:
@andrzejsydor 
Great observation! I chose JDK 17 because it's a long-term support (LTS) release, ensuring stability and extended support. Additionally, the project code is fully compatible with Java 17, so I opted for it.